gossip.asciidoc

```diff
@@ -644,68 +644,388 @@ channel graph) has the following attributes:
 ##### Channel Announcement Validation
 
 Now that we know what a `channel_announcement` contains. We can now move onto
-to exactly _how_ to verify such an announcement.
-
+to exactly _how_ to verify such an announcement. Verifying a channel
+announcement entails verifying that:
+
+ 1. The `chain_hash` specified by the channel announcement belongs to a known,
+ and supported blockchain.
+
+ 2. An _unspent_ UTXO exists in the chain as identified by the short channel
+ ID.
+
+ 3. The unspent UTXO has a p2wsh multi-sig script that is composed of
+ `bitcoin_key_1`, and `bitcoin_key_2`, ordered in lexicographic order.
+
+ 4. The signatures of `node_signature_1`, `node_signature_2`,
+ `bitcoin_signature_1`, and `bitcoin_signature_2` are valid signatures under
+ the corresponding public keys over the defined message digest.  The message
+ digest is the serialization of the raw message bytes after the final
+ signature. More on this in the chapter that covers wire message framing.
+
+The channel announcement is a core component of the Lighting Network channel
+graph, as it itself is an unauthenticated proofs of the existence of a channel.
+As verification of channel announcement ensures that the output is _unspent_ in
+order to add a new edge (a channel) to the public channel graph an upfront
+chain fee must be paid. In addition to this by verifying the signatures of both
+the bitcoin and node signatures, we ensure that the parties advertising the
+channel are able to update the underlying multi-sig of the channel, and also
+are aware of each other (all pubkeys are signed).
+
+A channel announcement is made available to the public network only for the set
```

Comment:
Maybe splitting into 

* creation of channel announcement message (and put that towards the end of the channel construction chapter) 
* validation as mostely done here and keep it in the gossip chapter